### PR TITLE
Dev4

### PR DESCRIPTION
**Pull request by kofimeighan**
Majority of the finishes touches for front-end.

### CODE REVIEW

File: portfolio/src/main/webapp/education.html [hunk 1]
```diff
@@ -8,28 +8,75 @@
...
+                   width=350px 
+                   alt="NSBE Logo"></span>
+        <p><i><b>Read Ahead</b></i> - I am a mentor with Read Ahead, 
+          an organization that connects adults and college students with
+          underpriviledged elementary and middle school kids in Morningside
+          Heights.</p>
+
+        To download my full resume, please click the icon
+          <a href="images/resume.pdf" download> 
+            <img src="images/download icon.png" alt="Resume Download"
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/education.html:77]: Ha, you missed formatting this one.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -34,11 +36,12 @@ <h3><b>Who am I?</b></h3>
...
         That was the start, and finally being able to apply my skills to 
         projects that will advance technology and positively develop 
-        someone's experience.
+        someone's experience is what continues to fuel me to learn and make 
+        change throughout the world.
 
         <img src="images/kofianddylan.jpg" 
-            height=350px class="center" class="border"
-            alt="Kofi & Dylan">
+             height=350px class="center" class="border"
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/index.html:43]: These two "class" should be in new line.

File: portfolio/src/main/webapp/pictures.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      <li><a href="mixes.html">Music Mixs</a></li>
+    </ul>
+
+  <p>In the past,I would use my freetime to take pictures on an old film camera
+    to capture some pictures of different things I was doing. I specialized
+    a lot in profile pictures but I would often take pictures of anything
+    I found interesting.</p>
+
+    <div class="image-thumbnails">
+      <a href=""><img src=""/></a>
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/pictures.html:34]: Did you forget to delete this?

File: portfolio/src/main/webapp/pictures.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/pictures.html:4]: lower case "utf".

File: portfolio/src/main/webapp/pictures.html (hunk 3 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/pictures.html:34]: Yes! Thank you for catching it.

File: portfolio/src/main/webapp/education.html (hunk 1 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/education.html:77]: Slightly confused on the formatting I missed? Can you elaborate?

**Pull request merged**